Makefile

```diff
@@ -109,7 +110,7 @@ include $(MAKE_SCRIPT_DIR)/$(OSFAMILY).mk
 include $(MAKE_SCRIPT_DIR)/tools.mk
 
 # Search path for sources
-VPATH           := $(SRC_DIR)
+VPATH           := $(SRC_DIR):$(LIB_MAIN_DIR)
```

Comment:
(having too many directories in VPATH may cause problems with name collisions)

Reply:
Done.